src/auth/auth.proto

```diff
@@ -105,9 +105,9 @@ message AuthenticateResponse {
 message WhoAmIRequest {}
 
 message WhoAmIResponse {
-  reserved 2,4;
+  reserved 2,3,4;
   string username = 1;
-  int64 ttl = 3 [(gogoproto.customname) = "TTL"];
+  google.protobuf.Timestamp expiration = 5;
```

Comment:
@actgardner  do we also check in the generated proto.go file?

Reply:
Yes, add the generated proto go files :)